docs/src/main/asciidoc/ide-tooling.adoc

```diff
@@ -0,0 +1,157 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Quarkus Tools in your favorite IDE
+
+include::./attributes.adoc[]
+
+[[project-creation]]
+== IDE Overview
+
+The following IDE's has support for the community developed Quarkus Tools:
+
+* https://marketplace.visualstudio.com/items?itemName=redhat.vscode-quarkus[Quarkus Tools for Visual Studio Code]
+* https://marketplace.eclipse.org/content/quarkus-tools[Quarkus Tools for Eclipse]
+* https://plugins.jetbrains.com/plugin/13234-quarkus/versions[Quarkus Tools for IntelliJ]
+* Quarkus Tools for Eclipse Che
+
+In addition IntelliJ have additional support for Quarkus in their Ultimate non-open source version.
+
+* https://www.jetbrains.com/help/idea/quarkus.html[IntelliJ Ultimate Edition built-in Quarkus support]
+
+The table below gives an overview of the current IDE's with links and a high-level overview of their features.
+
+:vscode-logo: https://simpleicons.org/icons/visualstudiocode.svg 
+:eclipse-logo: https://simpleicons.org/icons/eclipseide.svg
+:intellij-logo: https://simpleicons.org/icons/intellijidea.svg
+:che-logo: che-icon-dark.svg
+[cols="6*^", header]
+|===
+| .
+| image:{vscode-logo}[200,200]
+{empty} +
+VSCode Quarkus Tools
+| image:{eclipse-logo}[200,200]
+{empty} +
+Eclipse Quarkus Tools
+| image:{intellij-logo}[100,100]
+{empty} +
+IntelliJ Quarkus Tools
+| image:{intellij-logo}[100,100]
+{empty} +
+Intellij Ultimate
+| image:{che-logo}[100,100]
+{empty} +
+Eclipse Che
+
+|Description
+|Visual Studio Code extension to install using the marketplace  
+|Eclipse plugin to install into Eclipse using an updatesite
+|Intellij plugin that works in Intellij Community and Ultimate. Available from Marketplace.
+|Built-in Quarkus features available only in IntelliJ Ultimate
+|Built-in Quarkus features available in Eclipse Che incl. che.openshift.io.
+
+|Status
+|Stable
+|Alpha
+|Beta
+|Stable (since 2019.3)
+|Coming Soon!
```

Comment:
:‑/

Reply:
this is from the original ide post made in 2019 - I didn't change these as I don't know what label you put on them today. Are they all just Stable?

Reply:
We started packaging Che stable releases with a Quarkus devfile [since january 2019](https://github.com/eclipse/che-devfile-registry/pull/144).

Reply:
cool - but at the time I asked che pm for when and looked at che.openshift.io I was informed it was not available to use/consume. I've changed it to stable now.

Reply:
You are right. I think we had resources problems on openshift.io at the time and the Quarkus sample didn't work at all.